clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/model/GoogleScalingPolicy.groovy

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2015 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.google.model
+
+import groovy.transform.Canonical
+import groovy.transform.ToString
+
+@Canonical
+@ToString(includeNames = true)
+class GoogleScalingPolicy {
+  int minNumReplicas = 2
+  int maxNumReplicas = 2
+  int coolDownPeriodSec = 60
```

Comment:
Except for min/max, they all came from observing the behavior when configuring autoscaling via pantheon. At the time, I thought the defaults of min == 1 && max == 10 seemed too broad. But the defaults I chose (2 && 2) certainly couldn't be any narrower. Feel free to match the defaults behavior of the api or change to whatever you think is more appropriate. This only really affects calls from a script since these requests will be fully-specified when originating in the ui.